docs/src/components/SelectBox/Animation.tsx

```diff
@@ -0,0 +1,73 @@
+import React, {useState} from 'react';
+import {SelectBox, ThemeProvider, ThemeType, useTheme} from 'dooboo-ui';
+
+import type {FC} from 'react';
+import {View} from 'react-native';
+import styled from '@emotion/native';
+import {useFonts} from 'expo-font';
+
+const Container = styled.View`
+  flex: 1;
+  align-self: stretch;
+  background-color: ${({theme}) => theme.background};
+
+  align-items: center;
+  justify-content: center;
+`;
+
+const StyledText = styled.Text`
+  color: ${({theme}) => theme.text};
+  font-size: 16px;
+  margin: 20px;
+`;
+
+function SelectBoxStory(): React.ReactElement {
+  const data = ['Item1', 'Item2', 'Item3', 'Item4'];
+
+  const {
+    media: {isMobile},
+  } = useTheme();
+
+  const [selectedIndex, setSelectedIndex] = useState<number>(0);
+  const [count, setCount] = useState<number>(0);
```

Comment:
It is common belief that omitting type is better if it can be inferred by compiler.

So I suggest you to do like this. (Including other parts of codes)

```js
 const [selectedIndex, setSelectedIndex] = useState(0);
 const [count, setCount] = useState(0);
```

Compiler already know `selectedIndex`,  `count ` is number.

Reply:
Agree